content/docs/studio/user-guide/model-registry/what-is-a-model-registry.md

```diff
@@ -9,22 +9,59 @@ production, etc.) of your models on top of Git.
 https://www.youtube.com/watch?v=DYeVI-QrHGI
 
 Iterative Studio Model Registry is built on top of Git, and uses Iterative’s
-open-source Git-based tools [GTO] and [MLEM].
+open-source Git-based tools [GTO] and [MLEM] and [DVC].
```

Comment:
Minor: I'm still not sure we need to highlight the individual tools near the top of the intro page. Can we at least make clear that no prior knowledge of these is necessary?

Reply:
Right, removed this sentence.